scipy/stats/_entropy.py

```diff
@@ -86,18 +89,22 @@ def differential_entropy(
     window_length: Optional[int] = None,
     base: Optional[float] = None,
     axis: int = 0,
+    method: str = "vasicek",
```

Comment:
That does sound like a good option. One thought is rather than naming a method, we could make it `auto`, giving us flexibility to change the default over time. Also, the few plots suggest that we might make `auto` mean `van es` for very small samples (<10), `ebrahimi` for moderate samples (10~1000), and `vasicek` beyond that.

Reply:
I like this! But we should be clear then in the doc that the `auto` behaviour could break the compatibility.

Reply:
OK, done in 9efd2c626d5fd717a229b320e68128ec76acdf67